client/my-sites/hosting/sftp-card/index.js

```diff
@@ -115,13 +117,10 @@ const SftpCard = ( {
 
 		return (
 			<>
-				<span>{ translate( 'You must reset your password to view it.' ) }</span>
-				<Button
-					onClick={ resetPassword }
-					disabled={ isPasswordLoading }
-					busy={ isPasswordLoading }
-					compact
-				>
+				<p className="sftp-card__password-explainer">
+					{ translate( 'For security reasons, your password needs a reset to view' ) }
```

Comment:
This sentence seems cut off, could you improve the copy?

Reply:
https://github.com/Automattic/wp-calypso/pull/38243